cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsEmitter.java

```diff
@@ -33,35 +39,88 @@
   private static final Logger LOG = LoggerFactory.getLogger(AggregatedMetricsEmitter.class);
```

Comment:
javadoc says "A {@link co.cask.cdap.common.metrics.MetricsCollector}" - it is not true, can you fix that?